packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -86,6 +89,19 @@ const generateModalTitle = (formik_ref, my_profile_store, table_type, selected_a
     return localize('Sell {{ currency }}', { currency: selected_ad.account_currency });
 };
 
+const MarketRateChangeErrorModal = ({ is_open, closeModal, setShouldShowPopup }) => (
```

Comment:
Could we separate this modal out of the buy-sell-modal and make it its own component?